attestation-agent/attester/src/bin/evidence_getter.rs

```diff
@@ -49,8 +50,17 @@ async fn main() {
     }
 
     let evidence = attester
-        .primary_evidence(report_data)
+        .primary_evidence(report_data.clone())
         .await
         .expect("get evidence failed");
     println!("{evidence}");
+
+    let additional_evidence = attesters
+        .additional_evidence(report_data)
+        .await
+        .expect("get additional evidence failed");
+
+    for (tee, evidence) in additional_evidence {
+        println!("{tee:?} => {evidence}");
+    }
```

Comment:
One thing to note that this will change the behavior of this binary. With the original one we can
```shell
./evidence_getter | <handle-evidence>
```
While now it cannot. But this is ok for me as it is only a debugging tool.

Reply:
updated